solr/contrib/ltr/src/java/org/apache/solr/ltr/response/transform/LTRFeatureLoggerTransformerFactory.java

```diff
@@ -77,7 +77,7 @@
   private String fvCacheName;
   private String loggingModelName = DEFAULT_LOGGING_MODEL_NAME;
   private String defaultStore;
-  private String defaultFormat;
+  private FeatureLogger.FeatureFormat defaultFormat = FeatureLogger.FeatureFormat.DENSE;
```

Comment:
There is a setter (`setDefaultFormat`), the intention behind that setter is that the `solrconfig.xml` can configure a default and then the `fl=[features]` clause need not specify a `format` (unless it wishes to).